compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -558,10 +564,15 @@ abstract class GenJSCode extends plugins.PluginComponent
         if (isStaticModule(sym)) genModuleAccessorExports(sym)
         else genJSClassExports(sym)
 
+      val (jsClassCaptures, generatedConstructor) =
+        genJSClassCapturesAndConstructor(sym, constructorTrees.toList)
+
+      val jsSuperClass = jsClassCaptures.map(_.head.ref)
```

Comment:
It doesn't. It has other arguments for all the other captures of the class (including the `outer` params). But `GenJSCode` arranges things so that the capture of the super class always comes first.

Note that this is not a requirement of the IR. In fact in the IR nothing says you need a capture for your superclass (`ClassDef.jsSuperClass` could refer exclusively to top-level stuff). But our compiler will currently always generate things that way.